drivers/hd44780/Makefile.include

```diff
@@ -0,0 +1,3 @@
+ifneq (,$(filter hd44780,$(USEMODULE)))
+    USEMODULE_INCLUDES += $(RIOTBASE)/drivers/hd44780/include
```

Comment:
This changes the include directory and should get specific feedback from people maintaining this.

If the directory is modified like this, I would also like a static test that the `include` is in this directory, it would prevent issues when people copy paste or rename in the middle of a PR.